prediction-applier-extension/pom.xml

```diff
@@ -128,6 +137,8 @@
                             <artifactSet>
                                 <includes>
                                     <include>org.alfresco:alfresco-hxinsight-connector-common</include>
+                                    <include>org.alfresco:alfresco-hxinsight-connector-common-authentication</include>
+                                    <include>org.apache.httpcomponents.core5:httpcore5</include>
```

Comment:
imo we could also get rid of this dependency but do we need to?

Reply:
If it isn't provided by ACS then we can include it. The only risk is that ACS might start including it in the future.